bot/exts/evergreen/rps.py

```diff
@@ -0,0 +1,82 @@
+from random import choice
+
+from discord import Member, TextChannel
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+CHOICES = ['rock', 'paper', 'scissor']
+SHORT_CHOICES = ['r', 'p', 's']
+
+# Instead of putting bunch of conditions to check winner,
+# we can just manage this dictionary
+WINNER_DICT = {
+    'r': {
+        'r': 0,
+        'p': -1,
+        's': 1,
+    },
+    'p': {
+        'r': 1,
+        'p': 0,
+        's': -1,
+    },
+    's': {
+        'r': -1,
+        'p': 1,
+        's': 0,
+    }
+}
+
+
+class RPS(commands.Cog):
+    """Rock Paper Scissor. The Classic Game!"""
+
+    @staticmethod
+    def get_winner(action_one: str, action_two: str) -> int:
+        """Returns result of match from (-1, 0, 1) as (lost, tied, won)."""
+        return WINNER_DICT[action_one][action_two]
+
+    @staticmethod
+    def make_move() -> str:
+        """Returns random move for bot from CHOICES."""
+        return choice(CHOICES)
```

Comment:
Given that these functions are used only once, I don't think there's any point to separating them from where they're being called from.

Reply:
I have removed them in new commit. Thanks.